cdap-ui/app/cdap/components/RulesEngineHome/RulesTab/index.js

```diff
@@ -120,9 +120,12 @@ export default class RulesTab extends Component {
             null
         }
         {
-          rules.map(rule => {
+          rules.map((rule, i) => {
             return (
-              <Rule rule={rule} />
+              <Rule
+                rule={rule}
+                key={i}
```

Comment:
is the order of the array guaranteed to be consistent? for example if I add another rule, will it get added as the last item of the array? If not, setting the index as key might cause a problem. Might be safer to use `rule.id` (if it exists)